pkg/collector/corechecks/ebpf/probe/tcp_queue_length.go

```diff
@@ -145,3 +155,35 @@ func (t *TCPQueueLengthTracer) GetAndFlush() TCPQueueLengthStats {
 
 	return result
 }
+
+func loadTCPQueueLengthCOREProbe(cfg *ebpf.Config) (*TCPQueueLengthTracer, error) {
+	kv, err := kernel.HostVersion()
+	if err != nil {
+		return nil, fmt.Errorf("error detecting kernel version: %s", err)
+	}
+	if kv < kernel.VersionCode(4, 8, 0) {
```

Comment:
In general, I don't think we should assume a relationship between kernel version and features available, since they can be backported or unavailable because of Kconfig. It is better to test for the actual feature/helper.

I've started some of that work in https://github.com/DataDog/datadog-agent/pull/14685